plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1087,7 +1087,7 @@ public void testCaseInsensitivePredicate()
     }
 
     @Test
-    public void testArithmeticPredicatePushdown()
+    public void testArithmeticModuloPredicatePushdown()
```

Comment:
It's not only about modulo (comparisons, subtraction)

see also "// TODO add coverage for other arithmetic pushdowns" and the method end.

let's keep the old name